tools/webpack-make.js

```diff
@@ -1,3 +1,3 @@
-#!/bin/sh
-echo "This got renamed to ./build.js. Please see HACKING.md"
-exit 1
+#!/bin/env node
+console.error("This got renamed to ./build.js. Please see HACKING.md");
+process.exit(1);
```

Comment:
I'd be in favor of just dropping this. It's been long enough now, this was only meant to adjust our muscle memory.

Reply:
Agreed, but someone suggested switching it over. Let's drop it in March then :)